packages/atomic/src/components/result-template-components/atomic-result-sections/atomic-result-section-actions.tsx

```diff
@@ -0,0 +1,19 @@
+import {Element, Component} from '@stencil/core';
+import {containsVisualElement} from '../../../utils/utils';
+
+/**
+ * The `atomic-result-section-actions` element, when added to a result template,
+ * changes the style and position of its content to match specifications
+ * from the result list element.
```

Comment:
We can leave this for future PRs as the components are implemented, but it would be helpful to eventually explain what a particular section does. Is it a block element, or inline. What styles does it apply, what result template components should be placed as children, etc.